src/extension.ts

```diff
@@ -35,6 +36,8 @@ export async function activate(context: vscode.ExtensionContext) {
   addSubscriptions(context);
   await initializeTheStarterSetOfFavorites(context);
   await applyInitialConfiguration();
+
+  addLiveShareIntegration(context);
```

Comment:
Tried to keep the Live Share separated, this is the second of the `2` functions that are added to the `Peacock` code.